include/aws/crt/iot/DeviceDefender.h

```diff
@@ -0,0 +1,168 @@
+#pragma once
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+#include <aws/crt/io/EventLoopGroup.h>
+#include <aws/crt/mqtt/MqttClient.h>
+
+#include <aws/iotdevice/device_defender.h>
+
+namespace Aws
+{
+    namespace Crt
+    {
+
+        namespace Io
+        {
+            class EventLoopGroup;
+        }
+
+        namespace Mqtt
+        {
+            class MqttConnection;
+        }
+
+        namespace Iot
+        {
+
+            class DeviceDefenderV1ReportTask;
+            class DeviceDefenderV1ReportTaskBuilder;
+
+            /**
+             * Invoked upon DeviceDefender V1 task cancellation.
+             */
+            using OnDefenderV1TaskCancelledHandler = std::function<void(void *)>;
```

Comment:
Do we want to shove the V1 stuff in a V1 namespace and not pollute the names?

Reply:
Good point, I'll create a DeviceDefenderV1 namespace and move things under that.